org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -31,6 +31,9 @@
             <unit id="org.eclipse.equinox.sdk.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.jdt.source.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
+            <repository location="https://download.eclipse.org/eclipse/updates/4.19-I-builds/I20210217-1800/"/>
+        </location>
+        <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
```

Comment:
[4.19M3-I-build](https://download.eclipse.org/eclipse/downloads/drops4/S-4.19M3-202102171800/) contains the dependent formatting changes we made in the upstream jdt.core. But the I-build doesn't contain xtend and xtext sdks, so still need fetch xtend and xtext from the old release. Once 2021-03 release is ready, we can merge these two locations back into one.